columnphysics/icepack_itd.F90

```diff
@@ -1714,7 +1715,6 @@ subroutine icepack_init_itd(ncat, hin_max)
 #ifndef CESMCOUPLED
             hi_min = p1    ! minimum ice thickness allowed (m) for thermo
```

Comment:
Might be worth changing this as well so we don't get tripped up later on. In the future we'll probably swap `CESMCOUPLED` for `ACCESSCOUPLED`
```suggestion
            hi_min = p2    ! minimum ice thickness allowed (m) for thermo
```